src/views/Discussions/NewDiscussionView.tsx

```diff
@@ -59,14 +60,15 @@ const NewDiscussionView: FC<NewDiscussionViewProps> = ({ onPost }) => {
                 placeholder={t('components.new-discussion.description.placeholder')}
                 rows={10}
                 multiline
+                disabled={isSubmitting}
               />
             </Grid>
             <Grid item>
               <Button
                 variant="primary"
-                text={t('components.new-discussion.buttons.post')}
+                text={isSubmitting ? t('buttons.processing') : t('components.new-discussion.buttons.post')}
```

Comment:
'Processing' and 'Post' feel like a generic action to a button. Extract it to buttons instead.

Reply:
Post is the action. the real name is 'Post comment'